src/lib/storage/storage_manager.hpp

```diff
@@ -44,6 +44,6 @@ class StorageManager : private Noncopyable {
   StorageManager() {}
   StorageManager& operator=(StorageManager&&) = default;
 
-  // Implementation goes here
+  std::map<std::string, std::shared_ptr<Table>> _tables;
```

Comment:
There was this debate about using a unordered_map here?
I would be fine with map, cause it is more predictable, as we also don't really know how good our hash would be, and map is guaranteed O(log n)

Reply:
I think either selection is fine. I don't expect the table pointer lookup to ever be the bottleneck in any real life operation.